tests/unit/core/reducers/test_addons.js

```diff
@@ -85,10 +85,11 @@ describe(__filename, () => {
     const state = addons(undefined,
       loadAddons(createFetchAddonResult(theme).entities));
 
-    const expectedTheme = {
+    const expectedTheme = createInternalAddon({
```

Comment:
This should not use `createInternalAddon()`. This test is a bit weird but the intention of it was to manually recreate the theme addon to test that the mapper is doing what we expect it to. If you use `createInternalAddon()` then this test is pointless since it relies on that for the mapping. Instead of manually creating the entire theme addon, the test was using some spreads for the mapped properties that exactly match the original. Maybe it needs some comments to clarify this?